specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.0-Preview.1/TextAnalytics.json

```
@@ -0,0 +1,1252 @@
+          "type" : "string"
+        }, {
+          "name" : "showStats",
+          "in" : "query",
+          "description" : "(Optional) if set to true, response will contain input and document level statistics.",
+          "type" : "boolean"
+        }, {
+          "in" : "body",
+          "name" : "multiLanguageBatchInput",
+          "description" : "Collection of documents to analyze.",
+          "required": true,
+          "schema" : {
+            "$ref" : "#/definitions/MultiLanguageBatchInput"
+          }
+        } ],
+        "responses" : {
+          "200" : {
+            "description" : "A successful call results in a list of recognized entities with links to a well-known knowledge base returned for each valid document",
+            "examples" : {
+              "application/json" : {
+                "documents" : [ {
+                  "id" : "1",
+                  "entities" : [ {
+                    "name" : "Microsoft",
+                    "matches" : [ {
+                      "score" : 0.88,
+                      "text" : "MSFT",
+                      "offset" : 0,
+                      "length" : 4
+                    }, {
+                      "score" : 0.99,
+                      "text" : "Microsoft",
+                      "offset" : 25,
+                      "length" : 9
+                    } ],
+                    "language" : "en",
+                    "id" : "Microsoft",
+                    "url" : "https://en.wikipedia.org/wiki/Microsoft",
+                    "dataSource" : "Wikipedia"
+                  }, {
+                    "name" : "Windows 10",
+                    "matches" : [ {
+                      "score" : 0.88,
+                      "text" : "Windows 10",
+                      "offset" : 44,
+                      "length" : 10
+                    } ],
+                    "language" : "en",
+                    "id" : "Windows 10",
+                    "url" : "https://en.wikipedia.org/wiki/Windows_10",
+                    "dataSource" : "Wikipedia"
+                  } ]
+                }],
+                "errors" : [ ],
+                "modelVersion": "2019-10-01"
+              }
+            },
+            "schema" : {
+              "$ref" : "#/definitions/EntityLinkingResult"
+            }
+          },
+          "500" : {
+            "description" : "Error Response",
+            "schema" : {
+              "$ref" : "#/definitions/Error"
+            }
+          }
+        }
+      }
+    },
+    "/keyPhrases" : {
+      "post" : {
+        "summary" : "Key Phrases",
+        "description" : "The API returns a list of strings denoting the key phrases in the input text. See the &lt;a href=\"https://aka.ms/talangs\"&gt;Supported languages in Text Analytics API&lt;/a&gt; for the list of enabled languages.",
+        "operationId" : "/keyPhrases",
```

Use an appropriate name for the operationId. Same as above comment.
```suggestion
        "operationId" : "keyPhrases",
```